apps/frontend/src/pages/people/[username].tsx

```diff
@@ -473,27 +524,6 @@ const UserProfilePage: React.FC = () => {
               </>
             </div>
           )}
-
-          {connection?.comment?.note && (
-            <div className="flex flex-col gap-2 py-4 px-4">
-              <span className="text-sm font-semibold text-primary font-sans">
-                Your Note
-              </span>
-              <span className="text-sm text-secondary font-sans font-normal">
-                {connection?.comment?.note}
-              </span>
-            </div>
-          )}
-
-          {connection?.comment?.emoji && (
-            <div className="flex flex-col gap-2 py-4 px-4">
-              <span className="text-sm font-semibold text-primary font-sans">
-                Your Label
-              </span>
-
-              <p className="text-2xl mt-2">{connection?.comment?.emoji}</p>
-            </div>
-          )}
```

Comment:
Removed to reduce the chances the info from being leaked accidentally?

Reply:
yeah exactly